### DISCUSSION TITLE
DEMO测试通过search()100条数据耗时在300+ms

### DISCUSSION THREAD

**Discussion by Dux-King** [milvus-io/milvus#21213] [Q&A and General discussion] [answered]
通过下边操作search需要300ms以上是需要修改什么配置吗？
过程：
1使用docker-compose 启动demo 
https://github.com/milvus-io/milvus/releases/download/v2.2.0/milvus-standalone-docker-compose.yml
2.使用attu模拟了100条数据
![image](https://user-images.githubusercontent.com/24750282/207496935-af3ad1e6-7c9b-485d-9b37-5c65430f5a93.png)
![image](https://user-images.githubusercontent.com/24750282/207497186-8e3fc89d-a5c6-4452-b767-4644a58150c5.png)
![image](https://user-images.githubusercontent.com/24750282/207497342-3f1a3daf-9fae-4a9a-a72b-377e92bc8052.png)

**Comment by yhmo** [comment, marked as answer]
如果search参数consistency_level用的是“Strong”的话：
```
results = collection.search(
	data=[[0.1, 0.2]], 
	anns_field="book_intro", 
	param=search_params,
	limit=10, 
	expr=None,
	consistency_level="Strong"
)
```
改成Eventually
```
results = collection.search(
	data=[[0.1, 0.2]], 
	anns_field="book_intro", 
	param=search_params,
	limit=10, 
	expr=None,
	consistency_level="Eventually"
)
```

**Reply by Dux-King** [reply]
果然可以了，多谢
